lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -78,7 +78,7 @@
     content_type:
         description:
             - Specify if the disk is a data disk or ISO image
-        choices: ['data', 'iso']
+        choices: ['data', 'iso', 'hosted_engine', 'hosted_engine_sanlock', 'hosted_engine_metadata', 'hosted_engine_configuration']
```

Comment:
Please write in documentation that those parameters works since Ansible 2.8 and only with engine 4.3+?

Reply:
The Hosted Engine disk content types are available with Engine 4.3+ and Ansible 2.8